scripts/globals/pets/wyvern.lua

```diff
@@ -154,7 +154,7 @@ end;
 
 function onMobDeath(mob, player)
     local master = mob:getMaster();
-    local pet = player:getPet()
+    local pet = master:getPet()
```

Comment:
I think we should just be dropping that line and changing the next to 
```lua
local numLvls = mob:getLocalVar("level_Ups")
```
since in this context the "mob" is the wyvern.

Otherwise we are effectively getting master from the mob then get the mob from the master right after.